vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -152,7 +155,9 @@ class ZoomPlugin(VCPluginMixin, IndicoPlugin):
     def init(self):
         super().init()
         self.connect(signals.plugin.cli, self._extend_indico_cli)
-        self.connect(signals.event.times_changed, self._times_changed)
```

Comment:
I'm not sure if the new signals really handle everything needed... try e.g. moving a contribution to start before the event start time. This triggers an update of the event's start time, but I don't think it triggers `event.updated`.

So maybe better to keep using `times_changed`? AFAIK this one triggers for all time-related updates in events, contributions, etc.

Reply:
My comment was purely technical - we have signals that handle all date/time changes so IMHO using those is better than using generic "something changed" signals.

But yes, if we keep things in sync I'd be exact - after all the Zoom client typically notifies people about upcoming scheduled meetings...

Reply:
I see what you mean, makes sense.